assets/src/dashboard/app/api/apiProvider.js

```diff
@@ -57,6 +57,10 @@ export default function ApiProvider({ children }) {
 
   const fetchStories = useCallback(
     async ({ status = STORY_STATUSES[0].value }) => {
+      if (!api.stories) {
+        return [];
+      }
```

Comment:
Storybook uses the `ApiProvider` but doesn't pass in a config, so I've added these checks so we don't make fetch requests on undefined routes.